frontend/src/components/TheModalIdle.vue

```diff
@@ -1,6 +1,6 @@
 <template>
```

Comment:
is this component unused?

Reply:
Yeah, it's supposed to automatically detect session expiry and redirect automatically. Not sure if it worked too well in CCOF and haven't had a requirement for this. I'll drop it.